locales/bn/json.json

```diff
@@ -51,19 +51,19 @@
     "All resources loaded.": "সবগুলি রিসোর্স লোড করা হয়েছে।",
     "All rights reserved.": "সমস্ত অধিকার সংরক্ষিত।",
     "All subscription plan prices are excluding applicable VAT.": "সমস্ত সাবস্ক্রিপশন প্ল্যান মূল্য প্রযোজ্য ভ্যাট ব্যতীত।",
-    "All subscription plan prices include applicable VAT.": "সমস্ত সদস্যতা পরিকল্পনা মূল্য প্রযোজ্য VAT অন্তর্ভুক্ত.",
+    "All subscription plan prices include applicable VAT.": "সমস্ত সাবস্ক্রিপশন পরিকল্পনা মূল্য প্রযোজ্য VAT অন্তর্ভুক্ত.",
     "Already registered?": "ইতিমধ্যে নিবন্ধন করেছেন?",
     "American Samoa": "আমেরিকান সামোয়া",
     "Amount": "পরিমাণ",
-    "An error occured while uploading the file.": "ফাইল আপলোড করার সময় একটি ত্রুটির পাওয়া গিয়েছে।",
+    "An error occured while uploading the file.": "ফাইল আপলোড করার সময় একটি ত্রুটি পাওয়া গিয়েছে।",
```

Comment:
"ফাইল আপলোড করার সময় একটি ত্রুটি পাওয়া গিয়েছে।" should be changed to "ফাইলটি আপলোড করার সময় একটি ত্রুটি ঘটেছে।"